docs/source/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -924,17 +924,166 @@ intended distribution.
 2.3.9 Component 13: Periodic observations of attributes through survey, census, and registry
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+Census
+^^^^^^
+
+**When to Sample** 
+
+- The sample will be taken on the time step that includes April 1st of each decade (2010, 2020, 2030)
+- The sample will be taken on a single time steps 
+
+**What to Sample** 
+
+.. list-table:: Simulant Attribute to Sample 
+  :widths: 20
+  :header-rows: 0
+
+  * - Unique simulant ID (for PRL tracking)
+  * - First name
+  * - Middle initial 
+  * - Last name
+  * - Age 
+  * - Date of Birth 
+  * - Home Address 
+  * - Relationship to Person 1 (Head of Household)
+  * - Sex (binary)
+  * - Race/Ethnicity 
+
+**Who to Sample** 
+
+Based on race/ethnicity, age, and sex, simulants will be assigned a 
+probability of being missed in the census. Based on this 
+probability, simulants will be randomly selected for inclusion. 
+All data below is from the Census Post-Enumeration Survey. [Census_PES]_ 
+
+.. list-table:: Simulant Omission by Race/Ethnicity 
+  :widths: 20 10 10 
+  :header-rows: 1
+
+  * - Race/Ethnicity  
+    - Percent Omitted 
+    - Additive Risk of Omission (%)
+  * - US Total (all races) 
+    - 0.24
+    - 0 (reference)
+  * - White 
+    - -1.64
+    - -1.88
+  * - Black 
+    - 3.3
+    - 3.06
+  * - Asian 
+    - -2.62
+    - -2.86 
+  * - American Indian and Alaskan Native 
+    - 0.91
+    - 0.67
+  * - Native Hawaiian and Pacific Islander  
+    - -1.28
+    - -1.52 
+  * - Other Races or Multiracial  
+    - 4.34
+    - 4.1
+  * - Hispanic/Latino 
+    - 4.99
+    - 4.75 
+
+
+.. list-table:: Simulant Omission by Age/Sex 
+  :widths: 20 10 10 
+  :header-rows: 1
+
+  * - Age/Sex
+    - Percent Omitted 
+    - Additive Risk of Omission (%)
+  * - US Total (all ages and sexes) 
+    - 0.24
+    - 0 (reference)
+  * - 0-4, all sexes 
+    - 2.79
+    - 2.55
+  * - 5-9, all sexes 
+    - 0.1
+    - -0.14
+  * - 10-17, all sexes 
+    - 0.21
+    - -0.03
+  * - 18-29, male 
+    - 2.25
+    - 2.01
+  * - 18-29, female 
+    - 0.98
+    - 0.74
+  * - 30-49, male 
+    - 3.05
+    - 2.81
+  * - 30-49, female 
+    - -0.1
+    - -0.34
+  * - 50+, male 
+    - -0.55
+    - -0.79
+  * - 50+, female 
+    - -2.63
+    - -2.87
+
+Using the tables above, a probability of omission is calculated for 
+each simulant. The table below includes a few examples of this 
+process. 
+
+.. list-table:: Calculating Simulant Omission 
+  :widths: 10 10 10 20
+  :header-rows: 1
+
+  * - Simulant 
+    - Race/Ethnicity 
+    - Age/Sex
+    - Probability Risk of Omission (%)
```

Comment:
```suggestion
    - Probability of Omission (%)
```

I don't think this matches up very well with how we typically use the word "Risk."